test/HostActivationTests/GivenThatICareAboutStandaloneAppActivation.cs

```diff
@@ -154,20 +183,17 @@ private static void ReplaceTestProjectOutputHostInTestProjectFixture(TestProject
         {
             var dotnet = testProjectFixture.BuiltDotnet;
 
-            var testProjectHost = testProjectFixture.TestProject.AppExe;
             var testProjectHostPolicy = testProjectFixture.TestProject.HostPolicyDll;
             var testProjectHostFxr = testProjectFixture.TestProject.HostFxrDll;
 
-            if (!File.Exists(testProjectHost) || !File.Exists(testProjectHostPolicy))
+            if (!File.Exists(testProjectHostPolicy))
             {
                 throw new Exception("host or hostpolicy does not exist in test project output. Is this a standalone app?");
             }
 
-            var dotnetHost = Path.Combine(dotnet.GreatestVersionSharedFxPath, $"dotnet{testProjectFixture.ExeExtension}");
             var dotnetHostPolicy = Path.Combine(dotnet.GreatestVersionSharedFxPath, $"{testProjectFixture.SharedLibraryPrefix}hostpolicy{testProjectFixture.SharedLibraryExtension}");
-            var dotnetHostFxr = Path.Combine(dotnet.GreatestVersionSharedFxPath, $"{testProjectFixture.SharedLibraryPrefix}hostfxr{testProjectFixture.SharedLibraryExtension}");
+            var dotnetHostFxr = Path.Combine(dotnet.GreatestVersionHostFxrPath, $"{testProjectFixture.SharedLibraryPrefix}hostfxr{testProjectFixture.SharedLibraryExtension}");
```

Comment:
This whole method should be removed.  The referenced bug is closed because it was project.json specific. 

```
        /*
         * This method is needed to workaround dotnet build not placing the host from the package
         * graph in the build output.
         * [https://github.com/dotnet/cli/issues/2343](https://github.com/dotnet/cli/issues/2343)
         */
```

This DOES happen now, so this whole workaround should go away.